src/site/antora/modules/ROOT/examples/manual/filters/ContextMapFilter.json

```diff
@@ -0,0 +1,38 @@
+{
+  "Configuration": {
+    "monitorInterval": 10,
```

Comment:
Is `monitorInterval` required? If not, can we remove it from all newly added example configuration files, please?

Reply:
I intentionally added a `monitorInterval` attribute to all example configuration that use some variation of `ContextMapFilter`.

Basically each time I added a tip about `monitorInterval`, I also added the `monitorInterval` setting.